tests/test_download_puller.py

```diff
@@ -0,0 +1,112 @@
+import os
+import pytest
+import shutil
+import nbgitpuller.plugin_helper as ph
+import importlib
+import aiohttp
+from aioresponses import aioresponses
+google_nb = importlib.import_module("nbgitpuller.plugins.nbgitpuller-googledrive.googledrive_puller")
```

Comment:
I don't understand this line atm
- a more explicit variable name could help, what does google_nb represent?
- what motivates a dynamic import via a string / what does it mean to write like that? Could you add an inline comment?

Reply:
I guess it relates to that plugins are python modules that register themselves somehow, and even though the plugins folder are part of nbgitpuller folder structure, it doesn't mean it's installed as part of nbgitpuller or similar?

Reply:
yes and it should pulled out anyway especially if we put the plugins in their own projects with a separate PR